executor/distsql.go

```diff
@@ -414,6 +414,8 @@ type IndexLookUpExecutor struct {
 	// If dummy flag is set, this is not a real IndexLookUpReader, it just provides the KV ranges for UnionScan.
 	// Used by the temporary table, cached table.
 	dummy bool
+
+	partitionIDMap map[int64]struct{}
```

Comment:
Please add a comment here, that it will only be used for global indexes? I would also suggest to move it up, so it is just below prunedPartitions (or is it even possible to use prunePartitions instead of partitionIDMap?)

Reply:
all resolved.